services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> None:
+    return CinemaHall.objects.all()
+
+
+def create_cinema_hall(
+        hall_name: None,
+        hall_rows: None,
+        hall_seats_in_row: None

```

Comment:
The parameters `hall_name`, `hall_rows`, and `hall_seats_in_row` should have appropriate types instead of `None`. Consider using `str` for `hall_name` and `int` for `hall_rows` and `hall_seats_in_row`.